style.css

```diff
@@ -152,4 +155,47 @@ h1, h2, h3, h4, h5, h6 {
   =====================================
   Footer Bottom
   =====================================
-*/
\ No newline at end of file
+*/
+
```

Comment:
Looks good! I think maybe adding a  {font-weight: lighter}  to the Footer Links will give it that similar thin font! :) Other than that, it look perfect!

Reply:
Wow, yeah, that looks much better. Incorporated, gonna merge myself. Thanks!